04/1_string_case_insensitive_compare/main.c

```diff
@@ -0,0 +1,22 @@
+#include <stdio.h>
+
+#include "string.h"
+
+int main(void)
+{
+    const char STR0[] = "hello";
+    const char STR1[] = "HELL";
+    const char STR2[] = "yellow";
+    int DIFF;
+    
+    DIFF = string_case_insensitive_compare(STR0, STR1);
+    printf("hello compared to HELL: %d\n", DIFF);           /* 1 */

```

Comment:
on line 12.. you passed STR0 instead of "hello" and here you are hardcoding it... so there is no consistency.

either hardcode all.. or use that variable all the way..

for this one i suggest hardcode everything.. so remove STR1 STR2 STR0